packages/services/api/src/modules/organization/resolvers.ts

```diff
@@ -251,6 +253,22 @@ export const resolvers: OrganizationModule.Resolvers = {
         organization: organizationId,
       });
 
+      const currentUser = await injector.get(AuthManager).getCurrentUser();
+      const jsonUpdatedFields = JSON.stringify({
+        name: input.name
+      });
+      await injector.get(AuditLogManager).createLogAuditEvent({
+        eventType: 'ORGANIZATION_SETTINGS_UPDATED',
+        eventTime: new Date().toISOString(),
+        organizationId: organizationId,
+        userEmail: currentUser.email,
+        userId: currentUser.id,
+        metadata: {
+          readableMessage: `${currentUser.email} updated the organization name to ${input.name}`,
```

Comment:
I think we might not need `readableMessage`? the frontend can use all the fields to generate a view/message for it?

Reply:
agree!